protected/humhub/modules/marketplace/Module.php

```diff
@@ -71,10 +73,6 @@ public function getName()
         return Yii::t('MarketplaceModule.base', 'Marketplace');
     }
 
-    private $_onlineModuleManager = null;
-
-    private $_humhubApi = null;
-
```

Comment:
Funny place to locate field variables. :-) They have now been moved to the top ....